modules/tensor_mechanics/test/tests/1D_spherical/tests

```diff
@@ -6,6 +6,9 @@
     compiler = 'CLANG GCC'
     cli_args = 'Modules/TensorMechanics/Master/all/incremental=false
                 Materials/stress/type=ComputeLinearElasticStress'
+    issues = '#6256 7555'
+    design = 'source/Materials/ComputeRSphericalSmallStrain.md source/Materials/stress/type=ComputeLinearElasticStress.md syntax/Modules/TensorMechanics/Master/index.md'
+    requirement = 'The ComputeRSphericalSmallStrain class, called through the TensorMechanicsMaster action, shall find the total linearized solution for the displacement of a solid isotropic elastic sphere under an applied load on the outer radius of the sphere of -1MPa as $u(r) = \frac{- P * (1 - 2 * v) * r}{E}$ where P is the applied pressure, b is the outer radius, a is the inner radius, v is Poisson's ration, E is Young's Modulus, and r is the radial position.'
```

Comment:
This seems to me like we're getting a little too specific with the equation. I'd suggest something like this:

The mechanics system shall be capable of accurately computing the displacement of a thick-walled sphere with a pressure applied to the outer surface using a 1D spherical symmetric representation with elastic, total small strain assumptions.